packages/now-cli/src/util/dev/server.ts

```diff
@@ -784,7 +784,6 @@ export default class DevServer {
       this.getLocalEnv('.env.build', nowConfigBuild.env),
     ]);
     const allEnv = { ...buildEnv, ...runEnv };
-    Object.assign(process.env, allEnv);
```

Comment:
Why is this deleted? 🤔

Reply:
It's part of the bug. `vc dev`'s process.env should not be tainted. The project's configured env vars should be passed in to the child processes explicitly.

Consider that the "FOO" env var gets set in this parent process, and then the user deletes the `vercel.json` file so that "FOO" should no longer exist in the deployment, but it still does because it got assigned here in the parent process, which is wrong.

Reply:
I see, nice find! 👍